.changeset/tidy-pandas-live.md

```diff
@@ -0,0 +1,5 @@
+---
+'@faustwp/cli': patch
+---
+
+Added `@faustwp/blocks` version to telemetry data and include `wp-graphql-content-blocks` version from the WordPress telemetry endpoint.
```

Comment:
Suggesting this because there's another changeset with the content blocks info.

```suggestion
Added `@faustwp/blocks` version to telemetry data.
```